user_guide_src/source/installation/installing_composer.rst

```diff
@@ -131,8 +131,8 @@ to your project root
 Copy the ``env``, ``phpunit.xml.dist`` and ``spark`` files, from
 ``vendor/codeigniter4/framework`` to your project root
 
-You will have to adjust paths to refer to vendor/codeigniter/framework``, 
-- the $systemDirectory variable in ``app/Config/Paths.php``
+You will have to adjust the system path to refer to the vendor one, e.g. ``__DIR__ . '/../../vendor/codeigniter4/framework/system'``,
```

Comment:
using the following path is cleaner: 

ROOTPATH. '/vendor/codeigniter4/framework/system